src/main/java/edu/cornell/kfs/concur/rest/xmlObjects/ExpenseReportDetailsDTO.java

```diff
@@ -12,6 +12,9 @@
 @XmlAccessorType(XmlAccessType.NONE)
 public class ExpenseReportDetailsDTO {
     
+    @XmlElement(name = "ReportId")
+    private String reportId;
```

Comment:
This area uses "ReportId" and "reportId", but the ConcurReport class uses "reportID" instead. (The other XML DTO is also using "RequestID" and "requestID".) Does this DTO need to capitalize the "D" as well, or does Concur or KFS expect it to be just "Id" and not "ID"?